src/index.d.ts

```diff
@@ -4,7 +4,7 @@
  * git: https://github.com/sendbird/SendBird-UIKIT-JavaScript
  */
 import React from 'react';
-import Sendbird from 'sendbird';
+import Sendbird, { FileMessage, UserMessage } from 'sendbird';
```

Comment:
Can we use Sendbird. FileMessage , Sendbird.UserMessage, because everything else is written that way in this file